.rubocop.yml

```diff
@@ -0,0 +1,39 @@
+# use the shared YaST defaults
+inherit_from:
+  /usr/share/YaST2/data/devtools/data/rubocop_yast_style.yml
+
+# Redundant returns add legibility for developers used to other languages
+Style/RedundantReturn:
+  Enabled: false
```

Comment:
btw reason sounds a bit funny. Does we add to C++ also style that makes C++ more familiar for ruby developers? :)

Reply:
Just copied the rubocop config from what I considered the newest module and only added the exclude of old yast-firewall modules because of all the complains so. So will check the other options